RFS/src/main/java/com/rfs/RunRfsWorker.java

```diff
@@ -91,6 +97,14 @@ public static class Args {
 
         @Parameter(names = {"--log-level"}, description = "What log level you want.  Default: 'info'", required = false, converter = Logging.ArgsConverter.class)
         public Level logLevel = Level.INFO;
+
+
+        @Parameter(required = false,
+                names = {"--otelCollectorEndpoint"},
```

Comment:
Looks like we are using kabob casing

```suggestion
                names = {"--otel-collector-endpoint"},
```